core/backend-itwin-client/src/oidc/BackendAuthorizationClient.ts

```diff
@@ -71,6 +71,13 @@ export abstract class BackendAuthorizationClient extends ImsAuthorizationClient
     };
     const issuer = await this.getIssuer(requestContext);
     this._client = new issuer.Client(clientConfiguration);
+
+    custom.setHttpOptionsDefaults({
+      timeout: RequestGlobalOptions.timeout.response,
+      retry: RequestGlobalOptions.maxRetries,

```

Comment:
@paulius-valiunas does this address what you were looking to do in #731?

Reply:
ah, I forgot about that one :) yeah, that's what I wanted to do, except I'd also add it to `IntrospectionClient` and move it up above the `Issuer.discover` call in `TestBrowserAuthorizationClient`